src/elements/common/nav-button/NavButton.js

```diff
@@ -55,7 +55,7 @@ const NavButton = React.forwardRef<Props, React.Ref<any>>((props: Props, ref: Re
 
                             if (!event.defaultPrevented && isLeftClick(event)) {
                                 const method = replace ? history.replace : history.push;
-                                method(path);
+                                method(to);
```

Comment:
Should this still be `path`?

Reply:
Nope. I want to pass the entire location object, whether it's just a path or includes something like `state`.